internal/kafka/internal/workers/cluster_mgrs/dynamic_scaleup_mgr.go

```diff
@@ -57,14 +62,29 @@ func (m *DynamicScaleUpManager) Stop() {
 }
 
 func (m *DynamicScaleUpManager) Reconcile() []error {
+	var errs []error
 	if !m.DataplaneClusterConfig.IsDataPlaneAutoScalingEnabled() {
 		glog.Infoln("dynamic scaling is disabled. Dynamic scale up reconcile event skipped")
 		return nil
 	}
 	glog.Infoln("running dynamic scale up reconcile event")
 	defer m.logReconcileEventEnd()
 
-	errs := m.reconcileClustersForRegions()
+	// TODO remove this method call and the method itself the new dynamic scale up
+	// logic is ready
+	err := m.reconcileClustersForRegions()
+	if err != nil {
+		errs = append(errs, err...)
+	}
+
+	// TODO remove the "if false" condition once the new dynamic scale up
+	// logic is ready
+	if false {
```

Comment:
As discussed via chat with other members of the dynamic scaling work, @miguelsorianod  will work on a separate PR to:
- Remove the TODOs 
- Enable scale up with the new logic
- Update unit tests
- Update integration test to take into account the new logic

/cc @VanillaSpoon @JameelB